### PR TITLE
State handling and allow orientation change

### PR DESCRIPTION
**Pull request by cbalster**

### CODE REVIEW

File: app/src/main/java/de/stephanlindauer/criticalmaps/Main.java [hunk 1]
```diff
@@ -60,7 +63,17 @@ protected void onPostCreate(Bundle savedInstanceState) {
         drawerLayout.addDrawerListener(mDrawerToggle);
         mDrawerToggle.syncState();
         drawerLayout.addDrawerListener(new DrawerClosingDrawerLayoutListener());
-        navigateTo(R.id.navigation_map);
+
+        if(savedInstanceState != null) {
```
Comment (johnjohndoe) [inline app/src/main/java/de/stephanlindauer/criticalmaps/Main.java]: Missing whitespace after `if`

File: app/src/main/java/de/stephanlindauer/criticalmaps/Main.java [hunk 2]
```diff
@@ -159,6 +172,14 @@ public void onAttachedToWindow() {
     }
 
     @Override
+    protected void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
+
```
Comment (johnjohndoe) [inline app/src/main/java/de/stephanlindauer/criticalmaps/Main.java]: Unneeded whitespace.

File: app/src/main/java/de/stephanlindauer/criticalmaps/Main.java [hunk 3]
```diff
@@ -35,9 +35,12 @@
 
 public class Main extends AppCompatActivity implements NavigationView.OnNavigationItemSelectedListener {
 
+    private final static String KEY_NAVID = "main_navid";
```
Comment (johnjohndoe) [inline app/src/main/java/de/stephanlindauer/criticalmaps/Main.java]: You might want to prefix all keys with the package name which is available from the `BuildConfig.APPLICATION_ID + "main_navid`.

Comment (cbalster) [inline app/src/main/java/de/stephanlindauer/criticalmaps/Main.java]: thanks for the comments. will change this one later globally in separate commit.

**Pull request merged**